hub-stack

```diff
@@ -35,6 +35,8 @@ while [ "$1" != "" ]; do
                   ;;
     elaborate )   $HUB stack elaborate $@
                   ;;
+    remotedeploy ) hub-remote-deploy $@
```

Comment:
- [ ] hub-stack usage information missing.
- [ ] `hub-remote-deploy` is inconsistent with `$HUB stack elaborate`

Reply:
Well this is the question about where is job should be placed:
- are we replacing existing `hub stack deploy` or this would separate target?
- Should this job do only `deploy` or serve as wrapper for any command? 

Not sure what do you mean by `inconsistent`? 
For stack to be deployed we need to run elaborate either by user or by script.